webpack.config.js

```diff
@@ -0,0 +1,52 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+    entry: {
+        helloWorldPage: 'pages/hello-world/helloWorld.js'
+    },
+
+    output: {
+        path: path.resolve(__dirname, 'public'),
+        chunkFilename: 'webpack/[chunkhash].js',
+        filename: "javascripts/[name].js",
```

Comment:
If your app has multiple entry points, `[name]` maps to the [name of the chunk](https://webpack.js.org/concepts/output/#output-filename), which is the third parameter passed to [`require.ensure`](https://webpack.js.org/guides/code-splitting-require/#require-ensure-).

Since you aren't yet doing any code splitting, `[name]` just becomes an index number of the chunk (i.e. in this instance, the first chunk will be called `0.js`, not `helloWorldPage.js`). This may or may not be what you want.